libyul/AssemblyStack.cpp

```diff
@@ -94,8 +94,6 @@ void AssemblyStack::optimize()
 	if (!m_optimiserSettings.runYulOptimiser)
 		return;
 
-	if (m_language != Language::StrictAssembly)
-		solUnimplemented("Optimizer for both loose assembly and Yul is not yet implemented");
```

Comment:
Interesting. Does this mean we can now fuzz typed yul as well?

Reply:
In general, yes, but I don't think it makes any difference.